source/_components/notify.html5.markdown

```diff
@@ -29,6 +29,10 @@ notify:
     name: NOTIFIER_NAME
     gcm_api_key: YOUR_API_KEY
     gcm_sender_id: YOUR_SENDER_ID
+    #or
+    vapid_pub_key: YOUR_PUBLIC_KEY
```

Comment:
Maybe we should provide an additional example?
We like to keep example cut & pasteable....